packages/web/components/UserProfile/PageLink.js

```diff
@@ -22,25 +25,32 @@ PageLink.defaultProps = {
 };
 
 const SectionLink = styled.a`
-	${({ theme: { colors } }) => css`
+	${({ theme: { colors }, active }) => css`
 		display: flex;
 		align-items: center;
 		font-size: 1.6rem;
 		margin-bottom: 2rem;
 		color: ${colors.lightGray};
 		padding-left: 2rem;
 
-		:hover {
-			color: ${colors.darkGreen};
-		}
-
 		svg {
 			fill: ${colors.lightGray};
 			stroke: ${colors.lightGray};
 			width: 2rem;
 			height: 2rem;
 			margin-right: 1rem;
 		}
+
+		${active &&
+			css`
+				font-weight: bold;
+				color: ${colors.secondary};
+
+				svg {
+					fill: ${colors.secondary};
+					stroke: ${colors.secondary};
+				}
+			`}
 	`};
```

Comment:
I did some improvements to the active link css to match figma.

Reply:
Booa @luizeboli